mapproxy/config_template/base_config/full_example.yaml

```diff
@@ -425,10 +425,27 @@ sources:
     req:
       # username and password are extracted from the URL and do not show
       # up in log files
+      # ATTENTION: see https://github.com/mapproxy/mapproxy/issues/182
```

Comment:
please remove this line and move the information to the docs in the appropriate section